src/assessment.js

```diff
@@ -39,14 +46,14 @@ var q2 = [/* your answer here */];
 //
 //     `['alderaan', 'chartreuse', 'frickinLasers', 'laserSize']`
 //
-var q3 = [/* your answer here */];
+var q3 = ['laserSize'];
```

Comment:
`alderaan` is also defined, because it's outside of the `deathStar` scope while `fire` is inside `deathStar`. You can always see _out_; you can never see _in_.